packages/edit-post/src/components/text-editor/style.scss

```diff
@@ -15,8 +15,8 @@
 		}
 
 		// Hide the thick left border in the code editor.
-		&:not(.is-focus-mode):not(.has-fixed-toolbar):not(.is-selected) .editor-post-title__input:hover,
-		&:not(.is-focus-mode):not(.has-fixed-toolbar).is-selected .editor-post-title__input {
```

Comment:
This is not needed anymore since the title will no longer add the class. It's added to the visual editor component.